src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -64,12 +69,24 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     reloading: false,
     error: false,
     errors: [],
+    traceSize: '?',
```

Comment:
Should this be declared in the `State` type above? And any reason not to use an optional number type and format it accordingly instead of storing a `'?'`?